main.py

```diff
@@ -33,8 +33,59 @@ def create_new_offer(offers, products, customers):
     # Omogućite unos kupca
     # Izračunajte sub_total, tax i total
     # Dodajte novu ponudu u listu offers
-    pass
+    print("Dostupni kupci su:")
 
+    for i in range(len(customers)):
+        print(i+1, customers[i]["name"])
+
+    broj_kupca = int(input("Unesite broj koji se nalazi ispred zeljenog kupca:")) -1
+    kupac = customers[broj_kupca]["name"]
```

Comment:
Uzimaš samo ime kupca kao string i spremaš ga u "customer" polje, umjesto cijelog objekta kupca.

```suggestion
    kupac = customers[broj_kupca]
```